desktop/src/main/java/bisq/desktop/util/GUIUtil.java

```diff
@@ -1022,4 +1050,22 @@ public static void openTxInBsqBlockExplorer(String txId, Preferences preferences
         if (txId != null)
             GUIUtil.openWebPage(preferences.getBsqBlockChainExplorer().txUrl + txId, false);
     }
+
+    public static String getBsqInUsd(Price bsqPrice,
```

Comment:
We do have a mixture of in the codebase right now, but I would set BSQ always in uppercase to be able to differentiate it quicker from Bisq. Besides that utACK

Reply:
You mean getBSQInUsd instead of getBsqInUsd? In the DAO where it appears often we dont use uppercase. In general we dont use all uppercase (getApiKey not getAPIKey) for  abbreviation as the following Camelcase letter might make the abbreviation harder to read. Of course question of tase and opinon, but code base is follwing that style largely.

Reply:
I just wanted to point it out as I found both variations in the codebase and we should stick to one or the other.

Reply:
@sqrrm @freimair as maintainers what's your opinion on that so we have a rule going forward?

Reply:
Java is CamelCase, most of our code-base is CamelCase, why do something different here. Looks ugly both ways :)

Reply:
Ok, than lets leave it in camelCase.